### ISSUE TITLE
'utf-8' codec can't decode byte 0xb9 in position 1704: invalid start byte

### ISSUE THREAD

**Issue by CachCheng** [Zulko/moviepy#1073]
```
File "/root/anaconda3/lib/python3.7/site-packages/moviepy/video/io/VideoFileClip.py", line 91, in __init__
    fps_source=fps_source)
  File "/root/anaconda3/lib/python3.7/site-packages/moviepy/video/io/ffmpeg_reader.py", line 33, in __init__
    fps_source)
  File "/root/anaconda3/lib/python3.7/site-packages/moviepy/video/io/ffmpeg_reader.py", line 263, in ffmpeg_parse_infos
    infos = error.decode('utf8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb9 in position 1704: invalid start byte
```

Can you update your code with this error?

**Comment by tburrows13** [comment]
Does #169 help? Try running `ffmpeg -i test.mov -o test.avi` to convert your movie to something that moviepy definitely can use.

**Comment by tburrows13** [comment]
Fixed in #959.

**Comment by keikoro** [comment]
Closing this as the PR with the fix has been merged.
